packages/typography/package.json

```diff
@@ -1,10 +1,10 @@
 {
-  "name": "bl-typography",
+  "name": "@blaze-elements/typography",
   "version": "1.2.0",
   "main": "dist/main.js",
   "license": "MIT",
   "scripts": {
-    "start": "../../node_modules/.bin/webpack-dev-server --config ../../webpack.config.js --env.dev --hot --host 0.0.0.0 --env.element=typography",
+    "start": "../../node_modules/.bin/webpack-dev-server --config ../../webpack.config.js --env.dev --hot --env.element=typography",
```

Comment:
isn't the diff obvious? 

`--hot` is argument for webpack-dev-server
`--env.<foo>` is for webpack